### PR TITLE
Fix/refactoring

### PR DESCRIPTION
**Pull request by noahdurbin**
## Type Of Change
- [ ] New Feature
- [X] Debugging
- [X] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change

Added tests for methods not covered.

## Group Project Check List
- [X] I made test for my code, and made sure they were passing before submitting PR
- [X] I assigned at least 2 group members to review my PR

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -10,14 +10,10 @@ class Game
   def initialize(game_data)
     @game_id = game_data[:game_id].to_i
     @season = game_data[:season].to_s
-    @type = game_data[:type]
-    @date_time = game_data[:date_time].to_s
     @away_team_id = game_data[:away_team_id].to_i
```
Comment (austincarrjones) [inline lib/game.rb:13]: YAGNI, love it

File: spec/team_factory_spec.rb [hunk 2]
```diff
@@ -12,4 +12,18 @@
     expect(teams).to all be_a(Team)
   end
 
+  it 'can show all teams' do
```
Comment (austincarrjones) [inline spec/team_factory_spec.rb:15]: Thanks for this one! Totally forgot to make tests for these methods.

**Review by austincarrjones — APPROVED** [review]

File: spec/stat_tracker_spec.rb [hunk 3]
```diff
@@ -102,7 +102,7 @@
   end
 
   describe "#count_of_teams" do
-    it "returns the total number of teams in data" do
+    xit "returns the total number of teams in data" do
```
Comment (zach-bergman) [inline spec/stat_tracker_spec.rb:105]: Still not sure why these aren't passing.

Comment (austincarrjones) [reply spec/stat_tracker_spec.rb:105]: There seems to be a teardown issue. Not sure why.

**Review by zach-bergman — APPROVED** [review]
Looks great, final PR

**Pull request merged**